src/game_components/ActiveGame.js

```diff
@@ -0,0 +1,149 @@
+import { Input, Button } from "antd";
+import "antd/dist/antd.css";
+import React, { useState, useEffect } from "react";
+import database from "../database.js";
```

Comment:
with create-react-app's bundler, you actually can just reference js files without the extension, so `../database` and `./VotingPage`